modules/tensor_mechanics/doc/content/documentation/systems/UserObjects/CrystalPlasticityStateVarRateComponentVoce.md

```diff
@@ -0,0 +1,54 @@
+# Crystal Plasticity State Var Rate Component Voce
+
+!syntax description /UserObjects/CrystalPlasticityStateVarRateComponentVoce
+
+## Description
+
+This UserObjects, `CrystalPlasticityStateVarRateComponentVoce` is meant to be used within the user object base crystal plasticity framework as a Voce hardening rule.
+
+\begin{equation}
+\label{eq:self_hardening_rate}
+h^{\alpha} = h_0^{alpha}\left(1-\frac{\tau^{\alpha}_{CRSS}-\tau_0^{\alpha}}{\tau_{Sat}^{alpha}-\tau_0^{alpha}} \right)^{m^{\alpha}}
+\end{equation}
+
+with
+
+\begin{equation}
+\label{eq:slip_system_hardening_rate}
+\dot{\tau}^{\alpha}_{CRSS} = \sum_{\beta=1}^{N} \lvert \dot{\gamma}^\beta \rvert h^\beta q^{\alpha \beta}
+\end{equation}
+
+
+where: $ \tau^{\alpha}_{CRSS} $ is the current critical resolved shear stress, $\tau^{\alpha}_{0}$ is initial critical resolved shear stress, $\tau_{Sat}^{alpha} $ is the saturation resolved shear stress, $q^{\alpha \beta}$ is a matrix containing self and latent hardening coefficients, and $h_0^{\alpha}$ and m^\alpha are hardening parameters.
+
+Like for the others crystal plasticity routines values of all the parameters can be specified by groups of slip systems.
+
+This userobjects assumes that slip systems are provided in a well defined order and are grouped by planes (see  modules/tensor_mechanics/test/tests/cp_user_object/input_slip_sys_bcc48.txt for BCC crystals and modules/tensor_mechanics/test/tests/cp_user_object/input_slip_sys.txt for FCC)
+
+Seven variables need to be specified and one value is required for each group:
+\begin{itemize}
```

Comment:
this does not render properly. Try using markdown here:

```
- item1
- more stuff
- and so on
```

renders as 

- item1
- more stuff
- and so on

Reply:
Done, hopefully it renders. By the way, how do I check offline that everything renders properly before pushing updates?